tests/silent-payment.test.ts

```diff
@@ -1,27 +1,38 @@
-/* global it */
 import assert from "node:assert";
-import { SilentPayment } from "../index";
 import { ECPairFactory } from "ecpair";
-import ecc from "../noble_ecc";
-const bitcoin = require("bitcoinjs-lib");
-bitcoin.initEccLib(ecc);
+import { SilentPayment } from "../src";
+import ecc from "../src/noble_ecc";
+import jsonImput from "./data/sending_test_vectors.json";
```

Comment:
imput 
![image](https://github.com/Overtorment/SilentPayments/assets/1913337/ed4e06d9-ac3e-44d8-bec2-52e052eebe3c)

Reply:
this or eslint-ptsr